trunk/SUAVE/Analyses/Aerodynamics/AVL_Inviscid.py

```diff
@@ -308,6 +308,20 @@ def build_surrogate(self):
         CL_sur                           = np.zeros(np.shape(AoA_mesh))
         CD_sur                           = np.zeros(np.shape(AoA_mesh))
         
+
+        for jj in range(len(AoA_points)):
+            for ii in range(len(mach_points)):
+                CL_sur[ii,jj] = cl_surrogate.predict([np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]])])
+                CD_sur[ii,jj] = cd_surrogate.predict([np.array([AoA_mesh[ii,jj],mach_mesh[ii,jj]])])
+        
+        fig = plt.figure('Coefficient of Lift Surrogate Plot')    
+        plt_handle = plt.contourf(AoA_mesh/Units.deg,mach_mesh,CL_sur,levels=None)
+        cbar = plt.colorbar()
+        plt.scatter(xy[:,0]/Units.deg,xy[:,1])
+        plt.xlabel('Angle of Attack (deg)')
+        plt.ylabel('Mach Number')
+        cbar.ax.set_ylabel('Coefficient of Lift')
+
```

Comment:
Should this be added?  Would these plots be regressed?

Reply:
May have been a Git Merge issue. I'll take out